package.json

```diff
@@ -36,14 +36,20 @@
     "mongoose": "^5.4.19",
     "multiparty": "^4.2.1",
     "nodemon": "^1.18.10",
+    "react": "^16.8.6",
+    "react-csv": "^1.1.1",
+    "react-loading": "^2.0.3",
     "react-router-dom": "^5.0.0"
   },
   "devDependencies": {
     "concurrently": "^4.1.0",
     "eslint": "^5.15.3",
     "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^4.1.0",
```

Comment:
We don't want to add prettier to the project as it may get in the way of eslint. You can use prettier online manually to fix the styled component files.